File: app/src/main/java/com/bytedance/tiktok/fragment/FansFragment.java
package com.bytedance.tiktok.fragment;

import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;
import com.bytedance.tiktok.R;
import com.bytedance.tiktok.adapter.FansAdapter;
import com.bytedance.tiktok.base.BaseFragment;
import com.bytedance.tiktok.bean.DataCreate;
import com.bytedance.tiktok.bean.VideoBean;
import java.util.ArrayList;
import butterknife.BindView;

public class FansFragment extends BaseFragment {
    @BindView(R.id.recyclerview)
    RecyclerView recyclerView;
    private FansAdapter fansAdapter;
    private ArrayList<VideoBean.UserBean> userBeans = new ArrayList<>();

    @Override
    protected int setLayoutId() {
        return R.layout.fragment_fans;
    }

    @Override
    protected void init() {
        recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));

        userBeans.addAll(DataCreate.userList);
        fansAdapter = new FansAdapter(getContext(), userBeans);
        recyclerView.setAdapter(fansAdapter);
    }

}

File: app/src/main/java/com/bytedance/tiktok/fragment/FansFragment.kt
package com.bytedance.tiktok.fragment

import android.os.Bundle
import android.view.View
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.bytedance.tiktok.adapter.FansAdapter
import com.bytedance.tiktok.base.BaseBindingFragment
import com.bytedance.tiktok.bean.DataCreate
import com.bytedance.tiktok.databinding.FragmentFansBinding

class FansFragment : BaseBindingFragment<FragmentFansBinding>({FragmentFansBinding.inflate(it)}) {
    private var fansAdapter: FansAdapter? = null

    companion object {
        val recyclerViewPool = RecyclerView.RecycledViewPool()
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        init()
    }
    fun init() {
        binding.recyclerview.layoutManager = LinearLayoutManager(context)
        binding.recyclerview.setHasFixedSize(true)
        binding.recyclerview.setRecycledViewPool(recyclerViewPool)
        fansAdapter = FansAdapter()
        fansAdapter?.appendList(DataCreate.userList)
        binding.recyclerview.adapter = fansAdapter
    }
}